md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
```

Comment:
Would the etiquette doc be a could place for conflict resolution guidelines @mandric ? Or do you see it as separate?